src/game.cpp

```diff
@@ -6164,13 +6164,7 @@ void game::examine( const tripoint &examp )
 
     const optional_vpart_position vp = m.veh_at( examp );
     if( vp ) {
-        if( u.controlling_vehicle ) {
-            add_msg( m_info, _( "You can't do that while driving." ) );
-        } else if( abs( vp->vehicle().velocity ) > 0 ) {
-            add_msg( m_info, _( "You can't do that on a moving vehicle." ) );
-        } else {
-            Pickup::pick_up( examp, 0 );
-        }
+        Pickup::pick_up( examp, 0 );
```

Comment:
I would prefer to directly call `vehicle::interact_with()` here
```suggestion
        vp->vehicle().interact_with( p, vp->part_index() );
```

Reply:
I left it that way so you can still pick things up from e.g. the seat you're sitting on (since that's a container).

Reply:
from the same UI you'd normally benefit from with examine, to be clear